제이_지유진/2주차_미션/미션2/contextapi/src/component/ThemeContent.tsx

```diff
@@ -0,0 +1,31 @@
+import clsx from "clsx";
+import { THEME, useTheme } from "../context/ThemeProvider";
+
+function ThemeContent() {
+  const { theme } = useTheme();
+
+  const isLightMode = theme === THEME.LIGHT;
+  return (
+    <div
+      className={clsx(
```

Comment:
className={clsx(
        "p-4 h-dvh w-full",
        isLightMode ? "bg-white" : "bg-gray-800"
      )}
clsx를 사용하셨는데

```suggestion
    className={`p-4 h-dvh w-full",
        ${isLightMode ? "bg-white" : "bg-gray-800"}
      `}
```
처럼 사용하시면 외부 라이브러리 없이 삼항연산자로 조건에 따라 다르게 스타일링을 줄 수 있습니다! 무엇이 더 좋다!는 딱히 없지만(clsx 라이브러리 자체는 크기가 작아서 그렇습니다) 알아두시면 좋을 것 같아 알려드립니다!

Reply:
오 감사합니다!!!!